src/vsfilesystem.cpp

```diff
@@ -602,23 +602,31 @@ void InitDataDirectory()
     if ( !vegastrike_cwd.empty() ) {
         data_paths.push_back( vegastrike_cwd );
         data_paths.push_back( vegastrike_cwd+"/.." );
-        data_paths.push_back( vegastrike_cwd+"/../data4.x" );
-        data_paths.push_back( vegastrike_cwd+"/../../data4.x" );
+        // DELETE 4.x no longer used since .5 release obsolete, removal required.
+        // data_paths.push_back( vegastrike_cwd+"/../data4.x" );
+        // data_paths.push_back( vegastrike_cwd+"/../../data4.x" );
+        // data_paths.push_back( vegastrike_cwd+"/data4.x" );
         data_paths.push_back( vegastrike_cwd+"/../../data" );
-        data_paths.push_back( vegastrike_cwd+"/data4.x" );
         data_paths.push_back( vegastrike_cwd+"/data" );
         data_paths.push_back( vegastrike_cwd+"/../data" );
         data_paths.push_back( vegastrike_cwd+"/../Resources" );
+        data_paths.push_back( vegastrike_cwd+"/Assets-Production");
+        data_paths.push_back( vegastrike_cwd+"/../Assets-Production");
+        data_paths.push_back( vegastrike_cwd+"/../../Assets-Production");
     }
     data_paths.push_back( "." );
     data_paths.push_back( ".." );
-    //data_paths.push_back( "../data4.x" ); DELETE 4.x no longer used since .5 release obsolete, removal required.
+    // DELETE 4.x no longer used since .5 release obsolete, removal required.
+    //data_paths.push_back( "../data4.x" );
```

Comment:
Just remove all 4.x data entries and the comment

Reply:
OK, good. I was erring on the side of caution. I'm glad I have permission to remove these entirely.

Reply:
Done.